app/Http/Controllers/Three/PostsController.php

```diff
@@ -51,6 +51,42 @@ public function __construct(PostService $posts, SignupRepository $signups, PostT
     }
 
     /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(PostRequest $request)
```

Comment:
@DFurnes added the same code here but in its own `store` method! Figured this will soon be updated though with the customized northstar updates we want to make.